appveyor.yml

```diff
@@ -13,7 +13,7 @@ image: Visual Studio 2019
 environment:
   global:
     CYG_ROOT: C:\cygwin64
-    CYG_MIRROR: http://cygwin.mirror.constant.com
+    CYG_MIRROR: http://mirrors.sonic.net
```

Comment:
Should this be http://mirrors.sonic.net/cygwin/ or does it recognize that it needs to go to the cygwin subfolder?